docs/topics/review-reports.adoc

```diff
@@ -234,6 +234,15 @@ The dependencies graph may be adjusted by using any of the following.
 * Double clicking anywhere will return the entire graph to the default state.
 * Clicking on any item in the legend will enable or disable all items of the selected type. For instance, selecting the embedded WARs icon will disable all embedded WARs if these are enabled, and will enable these dependencies if they are disabled.
 
+=== Transactions report
+
+A Transactions Report displays the call stack, which executes operations on relational database tables. It supports Spring Data JPA and the traditional preparedStatement() method for SQL statement execution. It does not support ORM frameworks, such as Hibernate.

```

Comment:
Transaction report, not Transaction Report. In general, Red Hat uses lower-case for words like "report", even if the UI doesn't.